refocus-bdk-server.js

```diff
@@ -278,23 +278,24 @@ module.exports = (config) => {
    *
    * @param {Express} app - App stream so we can push events to the server
    * @param {String} token - Socket Token needed to connect to Refocus socket
+   * @param {String} botId - Id of the bot
    */
-  function refocusConnectSocket(app, token) {
+  function refocusConnectSocket(app, token, botId) {
     const opts = {
       reconnect: true,
       'reconnection delay': 10,
       transports: ['websocket'],
       upgrade: false,
-      extraHeaders: {
-        Authorization: token
-      }
+      query: {
+        id: botId,
```

Comment:
id: botName should work here

Reply:
I set it up on the server side to use the id. I think that's necessary because emitted objects will have botId and roomId, but won't know the names. Make sense?